.github/workflows/pr_checks.yml

```diff
@@ -7,7 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7 ]
+        python-version:
+          - 3.7
```

Comment:
I think we should get rid of 3.7 to save test time

Reply:
Tests are running parallel so we will not save any time I think snd as we support 3.7, 3.8 and 3.9 I think it's still worth to test it. We can think of drop support for 3.7 later on but if we don't have any issue with it I would keep support it.

Reply:
I think we should continue to execute unit tests on all versions of python we plan on supporting. We can't tell users we support a python version we don't test on.